src/Shared/Recruit.Vacancies.Client/Infrastructure/Services/EmployerAccountService.cs

```diff
@@ -23,9 +23,11 @@ public async Task<IEnumerable<string>> GetEmployerIdentifiersAsync(string userId
         {
             try
             {
-                var accounts = await _accountApiClient.GetUserAccounts(userId);
+                // var accounts = await _accountApiClient.GetUserAccounts(userId);
 
-                return accounts.Select(acc => acc.HashedAccountId);
+                // return accounts.Select(acc => acc.HashedAccountId);
+
+                return new List<string>{"MXD78R"};
```

Comment:
Oops!

Reply:
Yep ooops. Removed that file from commit.